NBA-Stats-Dashboard/src/index.css

```diff
@@ -0,0 +1,53 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.App {
+  background-color: beige;
+  min-height: 100vh;
```

Comment:
Consider using dvh (dynamic viewport height) instead of vh. It adjusts the component size dynamically to accommodate other UI elements, such as the mobile address bar.

Reply:
I have now added dvh (dynamic viewport height) instead of vh in cases where the browser supports it.